README.md

```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
 
 #### Parameters
 
-- ids: either a single value for a single id, or a literal for several id:`{ id1: value, id2: otherValue }`. All configured idField must be given a value.
+- identifier: either a single value for a single primaryKey field, or a literal if several fields:`{ id1: value, id2: otherValue }`. All configured primaryKey fields must be given a value.
```

Comment:
nope, it is the values that will be passed to the primaryKey fields, aka the identifier of a document.